plugins/outputs/redistimeseries/redists.go

```diff
@@ -0,0 +1,82 @@
+package redistimeseries
+
+import (
+	"github.com/go-redis/redis/v7"
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/outputs"
+)
+
+var sampleConfig = `
+  ## The address of the RedisTimeSeries server.
+  addr = "127.0.0.1:6379"
+
+  ## password to login Redis
+  # password = ""
```

Comment:
Please use `address` and also supply a `username` option. Furthermore, it would be nice to also support TLS from the start. Take a look at the `redis` input plugin which uses `tls.ClientConfig` from `plugins/common`.